src/common.js

```diff
@@ -146,6 +147,14 @@ module.exports = function setup(env) {
     }
   }
 
+  function extend (namespace, delimiter) {
+    if(typeof delimiter === 'undefined') {
+      delimiter = ':';
+    }
+
+    return createDebug(this.namespace + delimiter + namespace);
```

Comment:
You can remove the if check and just change the `+ delimiter +` to `+ (delimiter || ':') +`